test/integration/spec/rtcpeerconnection.js

```diff
@@ -1518,24 +1527,43 @@ function testSetDescription(sdpSemantics, local, signalingState, sdpType) {
 }
 
 function makeTest(options) {
+  // NOTE(mroberts): https://bugs.chromium.org/p/webrtc/issues/detail?id=9540
```

Comment:
I think you can delete this comment at this point now. The dummy SDP used to not include `a=mid` or `a=group:BUNDLE` attributes. It does now, and we don't much care about the behavior we tested before.